spec/koans.js

```diff
@@ -93,30 +96,30 @@ describe('a template string, is wrapped in ` (backticks) instead of \' or ". ',
   describe('can evaluate variables, which are wrapped in "${" and "}"', function() {
     
     it('e.g. a simple variable "${x}" just gets evaluated', function() {
-      let evaluated = `x=x`
-      //expect(evaluated).toBe('x=' + x);
+      let evaluated = `x=${x}`;
+      expect(evaluated).toBe('x=' + x);
     });
     
     it('multiple variables get evaluated too', function() {
-      var evaluated = `x+y`;
-      //expect(evaluated).toBe(x + '+' + y);
+      var evaluated = `${x}+${y}`;
+      expect(evaluated).toBe(x + '+' + y);
     });
     
   });
 
   describe('can evaluate any expression, wrapped inside "${...}"', function() {
     
     it('all inside "${...}" gets evaluated', function() {
-      var evaluated = Number(`x+y`);
-      //expect(evaluated).toBe(x+y);
+      var evaluated = Number(x+y);
+      expect(evaluated).toBe(x+y);
     });
     
     it('inside "${...}" can also be a function call', function() {
       function getSchool(){ 
         return 'Ironhack'; 
       }
-      var evaluated = `getSchool()`;
-      //expect(evaluated).toBe('Ironhack');
+      var evaluated = getSchool();
+      expect(evaluated).toBe('Ironhack');
```

Comment:
Using backticks it would look like this
```
function getSchool(){ 
    return 'Ironhack'; 
}
var evaluated = `${ getSchool() }`;
expect(evaluated).toBe('Ironhack');
```